Automattic/woocommerce-payments
# Sync phone changes with WooPay phone field

Fixes #9431
Fixes #9235

#### Changes proposed in this Pull Request








When removing the focus of the billing address phone field, automatically update the WooPay phone number. 

#### Testing instructions





* Enable WooPay.
* Add products to the cart.
* Access blocks checkout page.
* Check `Securely save my information for 1-click checkout`.
* Change the billing address phone number and remove focus.
* The WooPay phone field should get updated.
* Test on classic checkout.
* Test again with WooPay pre-checking enabled (WCPay DevTools > `Force the pre_check_save_my_info flag (aka Default Opt-in) in the account cache to be true`).

-------------------


**Post merge**
